lms/urls.py

```diff
@@ -964,7 +964,9 @@
 
 if settings.FEATURES.get('ENABLE_API_DOCS'):
     urlpatterns += [
-        url(r'^api-docs/$', get_swagger_view(title='LMS API')),
+        url(r'^swagger(?P<format>\.json|\.yaml)$', schema_view.without_ui(cache_timeout=0), name='schema-json'),
+        url(r'^swagger/$', schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui'),
+        url(r'^redoc/$', schema_view.with_ui('redoc', cache_timeout=0), name='schema-redoc'),
```

Comment:
This URL gives me an error:
![redoc-error](https://user-images.githubusercontent.com/3628148/58638943-eee92080-82c3-11e9-8d22-fba67fe4d34d.png)

Reply:
I'm not sure what redoc is supposed to do, honestly.  Is it something we want?

Reply:
I'm in the same boat as you. I've never heard of it.

Reply:
OK, I'm removing the /redoc/ URL, at least for now.

Reply:
Seems like a view used by a particular JS library to view API docs?:
https://github.com/Rebilly/ReDoc
If we're not using it, just remove?